challenges/dog-photo-gallery/script.js

```diff
@@ -0,0 +1,61 @@
+const gallery = document.createElement("div");
+gallery.classList.add("photo-gallery");
+const displayDogBtn = document.createElement("button");
+displayDogBtn.textContent = "Click to display random dog";
+const ul = document.createElement("ul");
+const loadMsgEl = document.createElement("div");
+loadMsgEl.classList.add("loading-msg");
+
+document.body.append(gallery, displayDogBtn, ul, loadMsgEl);
+
+class DogGallery {
+  constructor() {
+    this.dogData = {};
+    this.isFetching = false;
+  }
+
+  async fetchDogData() {
+    const url = "https://dog.ceo/api/breeds/image/random";
+    const response = await fetch(url);
+    if (!response.ok) {
+      throw new Error(`Response status: ${response.status}`);
```

Comment:
You've included error handling in your fetchDogData method, which is excellent. However, you might want to provide more user-friendly feedback in the UI when an error occurs, rather than just logging it to the console.

Reply:
will do.